subsys/net/l2/openthread/Kconfig

```diff
@@ -157,6 +157,12 @@ config MBEDTLS_PROMPTLESS
 	bool
 	default y if !CUSTOM_OPENTHREAD_SECURITY
 
+config OPENTHREAD_MBEDTLS_DEBUG
+	bool "Enable mbedtls logs"
+	select MBEDTLS
+	select MBEDTLS_DEBUG
+	select MBEDTLS_DEBUG_C
```

Comment:
MBEDTLS_DEBUG depends on MBEDTLS symbol, which is promptless in default openthread configuration. Because of that it can't be directly selected in a conf file. 
I see now MBEDTLS_DEBUG_C is ncs specific, maybe it's not the best place for this kind of config.